cmake/module/ProcessConfigurations.cmake

```diff
@@ -74,6 +74,7 @@ function(set_default_config config)
   if(is_multi_config)
     get_property(help_string CACHE CMAKE_CONFIGURATION_TYPES PROPERTY HELPSTRING)
     set(CMAKE_CONFIGURATION_TYPES "${all_configs}" CACHE STRING "${help_string}" FORCE)
+    set(CMAKE_TRY_COMPILE_CONFIGURATION "${config}" PARENT_SCOPE)
```

Comment:
Could you add a link to [this](https://gitlab.kitware.com/cmake/cmake/-/issues/19512) here? This is arguably a bug that will likely be fixed in the future.

Reply:
Thanks! Done.